### PR TITLE
i18n(fr-FR): Update `why-astro.mdx` and `contribute.mdx` v3 translation

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### What kind of changes does this PR include?
- Minor content fixes
- Updated content 

#### Description

Just update content of why-astro.mdx` with v3 of english doc

### CODE REVIEW

**Comment by McFlyPartages** [comment]
This is my first PR.

I'm not a GitHub king so don't hesitate to tell me what's wrong or what can be improved.

Thanks

**Comment by McFlyPartages** [comment]
Hy,

Sorry but i add `contribute.mdx`on the original push.

Sorry

File: src/content/docs/fr/concepts/why-astro.mdx [hunk 1]
```diff
@@ -8,37 +8,36 @@ i18nReady: true
...
+En revanche, la plupart des frameworks web modernes sont conçus pour construire des *applications web*. Ces frameworks fonctionnent mieux pour construire des expériences plus complexes, semblables à des applications dans le navigateur : tableaux de bord d'administration avec connexion, boîtes de réception, réseaux sociaux, listes de tâches, et même des applications natives comme [Figma](https://figma.com/) et [Ping](https://ping.gg/).
 
-C'est l'une des différences les plus importantes à comprendre concernant Astro. L'accent unique mis par Astro sur le contenu lui permet de faire des compromis et d'offrir des fonctionnalités de performance inégalées qui n'auraient pas de sens pour des frameworks web plus axés sur les applications.
+C'est l'une des différences les plus importantes à comprendre concernant Astro. a focalisation unique d'Astro sur le contenu lui permet de faire des compromis et d'offrir des performances inégalées qui n'auraient pas de sens pour des frameworks web plus orientée applications.
```
Comment (pioupia) [inline src/content/docs/fr/concepts/why-astro.mdx:28]:
```suggestion
C'est l'une des différences les plus importantes à comprendre concernant Astro. L'unique focalisation mise par Astro sur le contenu lui permet de faire des compromis et d'offrir des performances inégalées qui n'auraient pas de sens pour des frameworks web plus orientés applications.
```

File: src/content/docs/fr/concepts/why-astro.mdx [hunk 2]
```diff
@@ -8,37 +8,36 @@ i18nReady: true
...
 
 :::tip
-Si votre projet se situe dans le deuxième camp des "applications", Astro n'est peut-être pas le bon choix pour votre projet... **et ce n'est pas grave !** Consultez [Next.js](https://nextjs.org/) pour une alternative à Astro plus axée sur les applications.
+Si votre projet se situe dans le champ des "applications", Astro n'est peut-être pas le bon choix... **et ce n'est pas grave !** Consultez [Next.js](https://nextjs.org/) pour une alternative à Astro plus axée sur les applications.
```
Comment (pioupia) [inline src/content/docs/fr/concepts/why-astro.mdx:31]:
```suggestion
Si votre projet se situe dans le deuxième camp des "applications", Astro n'est peut-être pas le bon choix... **et ce n'est pas grave !** Consultez [Next.js](https://nextjs.org/) pour une alternative à Astro plus axée sur les applications.
```

File: src/content/docs/fr/concepts/why-astro.mdx [hunk 3]
```diff
@@ -8,37 +8,36 @@ i18nReady: true
...
 :::
 
-
-
 ## Server-first
 
-**Astro privilégie autant que possible le rendu côté serveur par rapport au rendu côté client**. C'est la même approche que les cadres traditionnels côté serveur - PHP, WordPress, Laravel, Ruby on Rails, etc. -- utilisent depuis des décennies. Mais vous n'avez pas besoin d'apprendre un deuxième langage côté serveur pour la débloquer. Avec Astro, tout se résume toujours à HTML, CSS et JavaScript (ou TypeScript, si vous préférez).
+**Astro privilégie autant que possible le rendu côté serveur par rapport au rendu côté client**. Il s'agit de la même approche que les frameworks traditionnels côté serveur -- PHP, WordPress, Laravel, Ruby on Rails, etc. -- utilisent depuis des décennies. Mais vous n'avez pas besoin d'apprendre un deuxième langage côté serveur pour la débloquer. Avec Astro, tout se résume toujours à HTML, CSS et JavaScript (ou TypeScript, si vous préférez).
```
Comment (pioupia) [inline src/content/docs/fr/concepts/why-astro.mdx:36]: We unlock the `rendu côté serveur`, no?
```suggestion
**Astro privilégie autant que possible le rendu côté serveur par rapport au rendu côté client**. Il s'agit de la même approche que les frameworks traditionnels côté serveur -- PHP, WordPress, Laravel, Ruby on Rails, etc. -- utilisent depuis des décennies. Mais vous n'avez pas besoin d'apprendre un deuxième langage côté serveur pour le débloquer. Avec Astro, tout se résume toujours à HTML, CSS et JavaScript (ou TypeScript, si vous préférez).
```

File: src/content/docs/fr/concepts/why-astro.mdx [hunk 4]
```diff
@@ -51,33 +50,32 @@ De bonnes performances sont toujours importantes, mais elles sont *spécialement
 - 850ms plus rapide → 7% de conversions en plus ([COOK](https://web.dev/why-speed-matters/))
 - Chaque seconde de ralentissement → 10 % d'utilisateurs en moins ([BBC](https://www.creativebloq.com/features/how-the-bbc-builds-websites-that-scale))
 
-Dans de nombreux frameworks web, il est facile de créer un site web qui a l'air génial pendant le développement, mais qui se charge douloureusement lentement une fois déployé. JavaScript est souvent le coupable, car les téléphones des utilisateurs et les appareils moins puissants atteignent rarement la vitesse de l'ordinateur portable d'un développeur.
+Dans de nombreux frameworks web, il est facile de créer un site web qui semble parfait pendant le développement, mais qui se charge douloureusement lentement une fois déployé. JavaScript est souvent le coupable, car les téléphones des utilisateurs et les appareils moins puissants atteignent rarement la vitesse de l'ordinateur portable d'un développeur.
```
Comment (pioupia) [inline src/content/docs/fr/concepts/why-astro.mdx:53]:
```suggestion
Dans de nombreux frameworks web, il est facile de créer un site web qui semble parfait pendant le développement, mais dont le chargement est terriblement lent une fois déployé. JavaScript est souvent le coupable, car les téléphones des utilisateurs et les appareils moins puissants atteignent rarement la vitesse de l'ordinateur portable d'un développeur.
```

File: src/content/docs/fr/concepts/why-astro.mdx [hunk 5]
```diff
@@ -51,33 +50,32 @@ De bonnes performances sont toujours importantes, mais elles sont *spécialement
...
+Si vous avez besoin de plus de contrôle, vous pouvez étendre Astro avec plus de [100+ intégrations](https://astro.build/integrations/) comme [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), et plus encore.[Connectez votre CMS préféré (/fr/guides/cms/) ou déployez sur votre hôte préféré (/fr/guides/deploy/) avec une seule commande.
```
Comment (pioupia) [inline src/content/docs/fr/concepts/why-astro.mdx:79]:
```suggestion
Si vous avez besoin de plus de contrôle, vous pouvez étendre Astro avec plus de [100+ intégrations](https://astro.build/integrations/) comme [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), et plus encore. [Connectez votre CMS préféré](/fr/guides/cms/) ou [déployez sur votre hôte préféré](/fr/guides/deploy/) avec une seule commande.
```

**Review by pioupia — CHANGES_REQUESTED** [review]
A few minor problems, but it's a very good start!

**Comment by McFlyPartages** [comment]
@pioupia thank you for your suggestions, I've just validated them (normally).

**Comment by McFlyPartages** [comment]
I've corrected a few syntax URLs in the contribte file

**Review by pioupia — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**